hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -5301,7 +5301,7 @@ private long replayRecoveredEditsForPaths(long minSeqIdForTheRegion, FileSystem
 
       long maxSeqId;
       String fileName = edits.getName();
-      maxSeqId = Math.abs(Long.parseLong(fileName));
+      maxSeqId = Math.abs(Long.parseLong(fileName.split("\\.")[0]));
```

Comment:
For files already with the `corrupted` suffix, shouldn't we just skip the whole replay attempt from this point onwards? We already know it will not work, so we can save some computation.